user-guide/dataminer_services/Catalog/Change_email_dataminer_services_account.md

```diff
@@ -57,7 +57,7 @@ To have your new account linked to the correct organization, after you have logg
 
 ### Transferring data on DataMiner Community (Dojo)
 
-Upon logging into Dojo with your new account, you will notice that a new account has been created. If you want to transfer the data from your old account, you will need to contact Skyline. However, note that if you opt for the transfer, your old account will be deleted.
+Upon logging onto Dojo with your new account, you will notice that a new account has been created. If you want to transfer the data from your old account, you will need to contact Skyline. However, note that if you opt for the transfer, your old account will be deleted.
```

Comment:
To be fully grammatically correct, I was left wondering if it should be “onto” and not “on to.” Based on the info you sent me earlier, I would expect the latter.

Reply:
Both are correct (see [https://www.dictionary.com/e/onto-vs-on-to/](https://www.dictionary.com/e/onto-vs-on-to/)), but we should indeed pick one or the other and not use both on the same page. I'll adjust the occurrences of "onto" to "on to", as that is what is currently used the most in our documentation.